itou/www/signup/tests.py

```diff
@@ -728,6 +728,146 @@ def test_create_user_prescriber_without_org(self):
         subject = mail.outbox[0].subject
         self.assertIn("Confirmez votre adresse e-mail pour la Plateforme", subject)
 
+    @mock.patch(
+        "itou.utils.apis.api_entreprise.EtablissementAPI.get", return_value=(ETABLISSEMENT_API_RESULT_MOCK, None)
+    )
+    @mock.patch("itou.utils.apis.geocoding.call_ban_geocoding_api", return_value=BAN_GEOCODING_API_RESULT_MOCK)
+    def test_create_user_prescriber_with_same_siret_and_different_kind(
+        self, mock_api_entreprise, mock_call_ban_geocoding_api
+    ):
+        """
+        A user can create a new prescriber organization with an existing SIRET number,
+        provided that:
+        * the kind of the new organization is different from the existing one <-
+        * there is no duplicate of the (kind, siret) pair
+
+        f.i.:
+        * user can't create 2 PLIE with the same SIRET
+        * user can create a PLIE and a ML with the same SIRET
+        """
+
+        # same SIRET as mock
+        siret = "26570134200148"
+        existing_org_with_siret = PrescriberOrganizationFactory()
+        existing_org_with_siret.siret = siret
+        existing_org_with_siret.kind = PrescriberOrganization.Kind.ML
+        existing_org_with_siret.save()
+
+        # Step 1: Does the user work  for PE?
```

Comment:
Tout petit pinaillage : il y a deux espaces entre `work` et `for`.

Reply:
Euh...
```
        # Step 1: Does the user work for PE?
```
sur mon fichier local ?!